plugins/stronghold/src/lib.rs

```diff
@@ -182,6 +185,16 @@ enum ProcedureDto {
         private_key: LocationDto,
         msg: String,
     },
+    GetEvmAddress {
+        #[serde(rename = "privateKey")]
+        private_key: LocationDto,
+    },
+    Secp256k1EcdsaSign {
+        flavor: Secp256k1EcdsaFlavor,
+        #[serde(rename = "privateKey")]
+        private_key: LocationDto,
+        msg: String,
+    },
```

Comment:
i couldn't try the changes yet but is this enum not missing a `Secp256k1Ecdsa` variant?